src/index.html

```diff
@@ -4,14 +4,175 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content=
+      "width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldPersonalInfo">
+        <legend class="legend">Personal information</legend>
+        <div class="field">
+          <label>
+            Surname:
+            <input type="text" name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label>
+            Name:
+            <input type="text"
+              name="name"
+              minlength="4"
+              maxlength="15"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label>
+            How old are You?
+            <input type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label>
+            Full date of birth:
+            <input type="date" name="dateOfBirth">
+          </label>
+        </div>
+
+        <div class="lastSetForm">
```

Comment:
No needs to add such classes as `lastSetForm`
Just use `not:(:last-child)` pseudo-element for your fields
```suggestion
        <div>
```